File: commands/Owner/exec.js
const { Command, util: { exec, codeBlock } } = require("klasa");

class Exec extends Command {
  constructor(...args) {
    super(...args, {
      aliases: ["execute"],
      description: "Execute commands in the terminal, use with EXTREME CAUTION.",
      guarded: true,
      permissionLevel: 10,
      usage: "<expression:code|expression:string>",
      quotedStringSupport: false
    });
  }

  async run(msg, [input]) {
    if(input.code) input = input.code;
    const result = await exec(input, { timeout: "timeout" in msg.flags ? Number(msg.flags.timeout) : 60000 })
      .catch(error => ({ stdout: null, stderr: error }));
    if("silent" in msg.flags) return;
    const split = msg.flags.split;
    if(split) {
      return msg.channel.send(result.stderr || result.stdout, { split: true, code: "prolog" });
    }
    const output = result.stdout ? `**\`OUTPUT\`**${codeBlock("prolog", result.stdout)}` : "";
    const outerr = result.stderr ? `**\`ERROR\`**${codeBlock("prolog", result.stderr)}` : "";
    return msg.sendMessage([output, outerr].join("\n"));
  }
}

module.exports = Exec;

File: commands/owner/exec.go
package owner

import (
  "bytes"
  "fmt"
  "os/exec"
  "github.com/sapphire-cord/sapphire"
)

// Executes shell commands.
// Usage: <command:string...>
func OwnerExec(ctx *sapphire.CommandContext) {
  var stdout bytes.Buffer
  var stderr bytes.Buffer
  cmd := exec.Command("bash", "-c", ctx.JoinedArgs())
  cmd.Stdout = &stdout
  cmd.Stderr = &stderr

  err := cmd.Run()
  stdoutstr := stdout.String()
  stderrstr := stderr.String()
  var out string = ""

  if stdoutstr != "" {
    out += fmt.Sprintf("**`STDOUT:`**```\n%s```\n", stdoutstr)
  }

  if stderrstr != "" {
    out += fmt.Sprintf("**`STDERR:`**```\n%s```\n", stderrstr)
  }

  if err != nil {
    out += fmt.Sprintf("**`ERROR:`** ```\n%s```", err)
  }

  if out == "" {
    out = "```No Output.```"
  }

  if len(out) > 2048 {
    ctx.Reply("Output too long.")
    return
  }
  ctx.Reply(out)
}
